src/layouts/Layout.jsx

```diff
@@ -17,7 +17,35 @@ const propTypes = {
       children: PropTypes.func,
       location: PropTypes.object
     };
-
+    const style = {
```

Comment:
We have moved to CSS Modules, checkout `src/layouts/Layout.module.css` to see how it works